src/azure/cli/commands/arm.py

```diff
@@ -160,11 +160,22 @@ def required_values_validator(namespace):
 
 APPLICATION.register(APPLICATION.COMMAND_TABLE_LOADED, add_id_parameters)
 
-def register_generic_update(name, getter, setter, factory=None, setter_arg_name='parameters', #pylint:disable=too-many-arguments
-                            simple_output_query=None, custom_handlers=None):
+def _get_child_property(parent, collection_name, item_name):
```

Comment:
>_get_child_property [](start = 4, length = 19)

this would probably be better renamed to something like _get_item since we're retrieving objects instead of properties